### DISCUSSION TITLE
Update log_format to include the version of Kong

### DISCUSSION THREAD

**Discussion by subnetmarco** [Kong/kong#7236] [Ideas and feature requests]
Update the syslog `log_format` property to include the version of Kong when an error is being submitted.

**Comment by thibaultcha** [comment]
(In `refactor/cli` the reports were rewritten. It also includes the version, so not sure)

**Comment by Tieske** [comment]
@thefosk can this be closed? considering the rewritten reports?

**Comment by subnetmarco** [comment]
This is not just about the reports, it's about the syslog notification. We would need to change the  error log format ([here maybe](https://github.com/nginx/nginx/blob/master/src/core/ngx_syslog.c) or [here](https://github.com/nginx/nginx/blob/master/src/core/ngx_log.c)) to print also the version.
